src/main/java/ru/timur/learning/configuration/SecurityWebApplicationInitializer.java

```diff
@@ -0,0 +1,6 @@
+package ru.timur.learning.configuration;
+
+import org.springframework.security.web.context.AbstractSecurityWebApplicationInitializer;
+
+public class SecurityWebApplicationInitializer extends AbstractSecurityWebApplicationInitializer {
```

Comment:
That's what was lacking for security to work - I lied the week before, everything should work now, even non-default stuff. 🌵